lore/settings.py

```diff
@@ -289,6 +291,13 @@ def get_var(name, default):
     },
 }
 
+REST_FRAMEWORK = {
```

Comment:
From my perspective it's fine forever. I just wanted to make sure devops was aware in case this could cause a problem in deployment that you and I wouldn't predict, like my use of a hard-coded index name in Haystack did.